tests/ReportbackApiTest.php

```diff
@@ -134,4 +134,204 @@ public function testUpdatingReportbackItem()
         $this->assertEquals('approved', $rbItem->status);
 
     }
+
+    /**
+     * Test posting a reportback item containing an unencoded old emoji
+     *
+     * @return void
+     */
+    public function testPostingReportbackWithNormalOldEmoji()
+    {
+        // Mock sending image to AWS.
+        $this->fileSystem->shouldReceive('put')->andReturn(true);
```

Comment:
The lines like this will probably have to be changed once #42 is approved. I figure it makes sense to get approval on that first so I only have to change these ones once.